e2e/tests/render/render.go

```diff
@@ -0,0 +1,88 @@
+package render
+
+import (
+	"bytes"
+	"os"
+	"sync"
+
+	"github.com/loft-sh/devspace/cmd"
+	"github.com/loft-sh/devspace/cmd/flags"
+	"github.com/loft-sh/devspace/e2e/framework"
+	"github.com/loft-sh/devspace/pkg/util/factory"
+	"github.com/onsi/ginkgo"
+)
+
+var _ = DevSpaceDescribe("build", func() {
+
+	initialDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	// create a new factory
+	var f factory.Factory
+
+	ginkgo.BeforeEach(func() {
+		f = framework.NewDefaultFactory()
+	})
+
+	// Test cases:
+
+	ginkgo.It("should render helm charts", func() {
+		tempDir, err := framework.CopyToTempDir("tests/render/testdata/helm")
+		framework.ExpectNoError(err)
+		defer framework.CleanupTempDir(initialDir, tempDir)
+
+		stdout := &Buffer{}
+		// create build command
+		renderCmd := &cmd.RenderCmd{
+			GlobalFlags: &flags.GlobalFlags{
+				NoWarn: true,
+			},
+			SkipPush: true,
+			Writer:   stdout,
+		}
+		err = renderCmd.Run(f)
+		framework.ExpectNoError(err)
```

Comment:
Would be great if you check the rendered yamls as well, you can use the `--silent` flag for this

Reply:
done